lib/checkPullRequestBranch.js

```diff
@@ -32,14 +32,20 @@ const checkBranch = async (context) => {
     // Comment on the pull request.
     const wiki = 'wiki'.link(
       'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
-      'instructions-for-making-a-code-change');
+        'instructions-for-making-a-code-change'
+    );
     const commentParams = context.repo({
       issue_number: pullRequest.number,
       body:
-        'Hi @' + prAuthor + ', PRs made from develop, release or test ' +
-        'branches are not allowed. So this PR is being closed. Please make ' +
-        'your changes in another branch and send in the PR. To learn more ' +
-        'about contributing to Oppia, take a look at our ' + wiki + '. Thanks!'
+        'Hi @' +
+        prAuthor +
+        ', PRs made from develop branch or from a branch ' +
+        'whose name is prefixed with develop, release or test are not ' +
+        'allowed. So this PR is being closed. Please make your changes in ' +
+        'another branch and send in the PR. To learn more about contributing ' +
+        'to Oppia, take a look at our ' +
+        wiki +
+        '. (Rule 1 specifically). Thanks!',
```

Comment:
Remove the fullstop before, it isn't required, right?

Reply:
Nope not needed.
Removed.